README.md

```diff
@@ -2,56 +2,14 @@ athena
 ======
 <!-- Jenkins Status Badge in Markdown (with view), unprotected, flat style -->
 <!-- In general, need to be on Princeton VPN, logged into Princeton CAS, with ViewStatus access to Jenkins instance to click on unprotected Build Status Badge, but server is configured to whitelist GitHub -->
-<!-- [![Jenkins Build Status](https://jenkins.princeton.edu/buildStatus/icon?job=athena/PrincetonUniversity_athena_jenkins_master)](https://jenkins.princeton.edu/job/athena/job/PrincetonUniversity_athena_jenkins_master/) -->
```

Comment:
revert almost all changes in this file (I dont think you meant to change anything, right @yanfeij ?)
except below comment

Reply:
replace with the file from master